balancer/rls/picker.go

```diff
@@ -61,12 +61,13 @@ type rlsPicker struct {
 
 	// The picker is given its own copy of the below fields from the RLS LB policy
 	// to avoid having to grab the mutex on the latter.
-	defaultPolicy *childPolicyWrapper // Child policy for the default target.
-	ctrlCh        *controlChannel     // Control channel to the RLS server.
-	maxAge        time.Duration       // Cache max age from LB config.
-	staleAge      time.Duration       // Cache stale age from LB config.
-	bg            exitIdler
-	logger        *internalgrpclog.PrefixLogger
+	rlsServerTarget string
```

Comment:
Yeah I initially had that but I figured since this already persists bOpts around just use that. I guess code cleanliness it worth it in this case because it's only an extra string and pointer persisted around.

Reply:
Similar to how you store endpoint as well rather than access through balancer's Build options.